### PR TITLE
Corrected typos observed by #38

### PR DESCRIPTION
**Pull request by I-Bouros**
Corrected typos observed by @rcw5890 in #38

### CODE REVIEW

File: branchpro/models.py [hunk 1]
```diff
@@ -80,11 +80,12 @@ class BranchProModel(ForwardModel):
...
     simulate: return model output for specified parameters and times.
-    get_serial_intervals: returns serial intevals for the model.
-    update_serial_intevals: updates serial intevals for the model.
+    get_r_profile: returns R_t profile for the model.
+    get_serial_intervals: returns serial intervals for the model.
+    update_serial_intervals: updates serial intervals for the model.
     set_r_profile: creates a new R_t profile for the model.
 
-    *Always apply method set_r_profile before simulation for a change of R_t
+    *Always apply method set_r_profile before simulate for a change of R_t
```
Comment (Benjamin-Lee) [inline branchpro/models.py:88]:
```suggestion
*Always apply method `set_r_profile` before simulating for a change of R_t
```

Comment (I-Bouros) [reply branchpro/models.py:88]: Maybe I can use the quotation for simulate? I mean, simulate is a method for the class. What do you think?

Comment (Benjamin-Lee) [reply branchpro/models.py:88]: Ah I didn't catch that you were referring to the method. I think you can do this:

```suggestion
    *Always apply method `set_r_profile` before calling :meth:`BranchProModel.simulate` for a change of R_t
```

The `:meth:` directive will render it to a link to the simulate method in the docs

**Review by rcw5890 — APPROVED** [review]
Thanks for you prompt fix @I-Bouros ! :smile: 

Nice catch @Benjamin-Lee 
My sphinx didn't like `:meth:` inside those * so I just fixed that

**Pull request merged**